src/main/java/org/sagebionetworks/web/client/jsinterop/reactquery/InvalidateQueryFilters.java

```diff
@@ -0,0 +1,18 @@
+package org.sagebionetworks.web.client.jsinterop.reactquery;
+
+import jsinterop.annotations.JsOverlay;
+import jsinterop.annotations.JsPackage;
+import jsinterop.annotations.JsType;
+
+@JsType(isNative = true, namespace = JsPackage.GLOBAL, name = "Object")
+public class InvalidateQueryFilters {
```

Comment:
Breaking change in react-query means we need to put our `queryKey` in an `InvalidateQueryFilters` object when we invalidate queries.